examples/ensemble/plot_stack_predictors.py

```diff
@@ -60,35 +144,79 @@ def plot_regression_results(ax, y_true, y_pred, title, scores, elapsed_time):
 #
 # Here, we combine 3 learners (linear and non-linear) and use a ridge regressor
 # to combine their outputs together.
+#
+# Note: although we will make new pipelines with the processors which we wrote
+# in the previous section for the 3 learners, the final estimator RidgeCV()
+# does not need preprocessing of the data as it will be fed with the already
+# preprocessed output from the 3 learners.
 
-from sklearn.ensemble import StackingRegressor
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.experimental import enable_hist_gradient_boosting  # noqa
+
+from sklearn.experimental import enable_hist_gradient_boosting
 from sklearn.ensemble import HistGradientBoostingRegressor
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.ensemble import StackingRegressor
 from sklearn.linear_model import LassoCV
 from sklearn.linear_model import RidgeCV
 
-estimators = [
-    ('Random Forest', RandomForestRegressor(random_state=42)),
-    ('Lasso', LassoCV()),
-    ('Gradient Boosting', HistGradientBoostingRegressor(random_state=0))
-]
-stacking_regressor = StackingRegressor(
-    estimators=estimators, final_estimator=RidgeCV()
-)
 
+lasso_pipe = make_pipeline(processor_lin,
```

Comment:
I prefer to use `lasso_pipeline` instead of `lasso_pipe`. Because:

![Ceci n'est pas une pipe](https://upload.wikimedia.org/wikipedia/en/b/b9/MagrittePipe.jpg)